peachjam/templates/peachjam/_document_content.html

```diff
@@ -149,14 +152,21 @@
         {% if display_type == 'pdf' %}
           <div class="content" data-document-element>
             <div class="pdf-loading">
-              <div class="text-center mb-2"><strong>Loading PDF...</strong></div>
+              <div class="text-center mb-2"><strong>{% trans 'Loading PDF...' %}</strong></div>
               <div class="progress">
                 <div class="progress-bar bg-primary progress-bar-striped progress-bar-animated"
                      role="progressbar">
                 </div>
               </div>
             </div>
+
             <div class="pdf-content"></div>
+            <div class="pdf-large-prompt">
+              <p>{% trans 'This current document is more than 5MB. Proceed to load?' %}</p>
```

Comment:
This should tell the user how big it is -- 100MB? 5.1MB? I think this would be clearer:

"This document is 52MB. Do you want to load it?"

Reply:
refactored